sphinx/writers/latex.py

```diff
@@ -394,20 +396,32 @@ def __init__(self, document, builder):
             # no need for \\noindent here, used in flushright
             self.elements['logo'] = '\\sphinxincludegraphics{%s}\\par' % \
                                     path.basename(builder.config.latex_logo)
-        # setup multilingual package
+
+        # move fncychap setting to outside of babel settings (they are unrelated)
```

Comment:
oops, this comment is only made to describe my idea to you.
this does not describe how the code works.